README.md

```diff
@@ -1,158 +1,270 @@
+
+  <p><strong>Step 1:</strong> Using your Ewon device’s webpage, open the user account settings by selecting <code>Setup</code> &gt; <code>Users</code> in the menu on the left.</p>
+  <img src="images/ewonsetup-step1.png" width="173" height="203" />
+  <br />
+
+  <p><strong>Step 2:</strong> Create a new user account with the following information. Do not modify the user rights.</p>
+  <p><code>User Login</code>: <em>demo</em><br />
+  <code>Password</code>: <em>demopassword</em><br />
+  <code>Confirm Password</code>: <em>demopassword</em></p>
+  <p><strong>NOTE:</strong> This user account is required for Ewon Ignition Demo to function and allows the demo to configure tags on your Ewon device.</p>
+  <img src="images/ewonsetup-step2.png" width="440" height="130" />
+  <br />
+
+  <p><strong>Step 3:</strong> Open an FTP connection to your Ewon device using your operating system’s file manager (i.e. File Explorer).</p>
+  <p>To do this, type the following address in the address bar, replacing “[EWON IP HERE]” with the IP address you use to access your Ewon device’s webpage, then hit Enter/Return. <em>If required, enter the username/password you use to access your Ewon device’s webpage.</em></p>
+  <p><code>Address</code>: <em>ftp://[EWON IP HERE]</em></p>
+  <img src="images/ewonsetup-step3.png" width="513" height="54" />
+  <br />
+
+  <p><strong>Step 4:</strong> Using the created FTP connection, copy the files <code>EwonIgnitionDemoApp.jar</code> and <code>jvmrun</code> from the demo resource pack to the <code>usr</code> folder on your Ewon device.</p>
+  <img src="images/ewonsetup-step4.png" width="504" height="264" />
+  <br />
+
+  <p><strong>Step 5:</strong> Reboot your Ewon device by disconnecting and reconnecting power, or by selecting <code>Setup</code> &gt; <code>Reboot</code> on your Ewon device’s webpage.</p>
+  <img src="images/ewonsetup-step5.png" width="159" height="222" />
+
+  <!-- INSTALLATION/IGNITION SETUP -->
+  <h3 id="ignition-setup"><span style="font-variant: small-caps;">Ignition Setup</span></h3>
+  <p><em>During Ignition Setup, you’ll need access to your Ignition gateway’s webpage. For information on installing and accessing Ignition, please refer to <a href="#installing-and-configuring-ignition">Appendix 6.4, Installing and Configuring Ignition</a>.</em></p>
+
+  <p><strong>Step 1:</strong> Download and install the latest version of the Ignition Ewon Connector from the IgnitionEwonConnector GitHub repository, found here: <a href="https://github.com/hms-networks/IgnitionEwonConnector" class="uri">https://github.com/hms-networks/IgnitionEwonConnector</a>.</p>
+  <p>For instructions on how to download and install the connector, please refer to the “Download”, “Installation” and “Ewon Setup” sections of the Ignition Ewon Connector GitHub repository.</p>
```

Comment:
maybe just say refer to the readme.  If we change those heading names we would have to change this readme to match.  If you just say refer to the instructions in the readme there is no added maintenance.

Reply:
I agree, and I'd like to make each readme/instruction guide as separate and independent as possible, but for some reason, I envision that people might get confused here if we just toss them into the readme as a whole, since theres a lot more to it that they don't really need to be concerned with at this time in their setup process.